pkg/data/accounts.go

```diff
@@ -54,6 +55,7 @@ func (a *Account) queryAccounts(query *account.Account, keyName string, index st
 
 	res, err = a.DynamoDB.Query(queryInput)
 	if err != nil {
+		log.Println("err: ", err)
```

Comment:
Is this meant to be a permanent addition to the logging or was this added for debugging? If it is meant to be permanent, please add some context to the logger statement (e.g. "Error querying XYZ")